blocks/library/image/block.js

```diff
@@ -252,8 +252,8 @@ class ImageBlock extends Component {
 						return (
 							<ResizableBox
 								size={ {
-									width: currentWidth,
-									height: currentHeight,
+									width,
+									height,
 								} }
```

Comment:
It probably doesn't matter in practice but [the docs](https://github.com/bokuweb/re-resizable#size--width-number--string-height-number--string-) say that `size` is optional and that `.width` and `.height` are required. 

```js
<ResizableBox
    size={ width && height && {
        width,
        height,
    } }
    ...
/>
```